apps/aws-amplify/src/locations/Sidebar.tsx

```diff
@@ -1,17 +1,59 @@
-import React from 'react';
-import { Paragraph } from '@contentful/f36-components';
+import { Paragraph, Button } from '@contentful/f36-components';
 import { SidebarAppSDK } from '@contentful/app-sdk';
 import { /* useCMA, */ useSDK } from '@contentful/react-apps-toolkit';
+import Axios, { AxiosError } from 'axios';
+import { useState } from 'react';
 
 const Sidebar = () => {
   const sdk = useSDK<SidebarAppSDK>();
+  const errorText = 'The AWS website build was not triggered. Try again later.';
+  const [isLoading, setIsLoading] = useState(false);
   /*
      To use the cma, inject it as follows.
      If it is not needed, you can remove the next line.
   */
   // const cma = useCMA();
 
-  return <Paragraph>Hello Sidebar Component (AppId: {sdk.ids.app})</Paragraph>;
+  const buildAmplifyApp = () => {
+    setIsLoading(true);
+    const webhookURL = 'https://sdlb4gygrnzr252nrx4h3tghym0fameu.lambda-url.us-west-2.on.aws/';
```

Comment:
💪 nice! I know this probably a temp, maybe we shove move into an ENV var? Can do that as cleanup in a "final pass" fast follow with the other small things